src/cpp/rtps/builtin/discovery/participant/PDPClient.cpp

```diff
@@ -26,7 +26,9 @@
 #include <string>
 #include <tuple>
 #include <vector>
+#include <vector>
 
+#include <fastdds/builtin/type_lookup_service/TypeLookupManager.hpp>
```

Comment:
This header is internal and should be in the internal header block.

Reply:
moved in [d259c5d](https://github.com/eProsima/Fast-DDS/pull/4240/commits/d259c5dfab253e283b1f3647dbe12ebf9e79ca2f)